index.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /*
-Plugin Name: WooCommerce Komoju Gateway
+Plugin Name: Komoju Payments

```

Comment:
Renaming to match the name of the plugin. From reading through the requirements I don't think we can call this "WooCommerce..." since that's not our copyright. I picked `Komoju Payments` because that's how the magento plugin is referred to, but happy to change if needed.

Reply:
We should change Komoju to KOMOJU here - also I'm thinking we don't need Payments in the name unless we plan on adding more woocommerce plugins under the brand

Reply:
Yup, I've updated it in f627ee6 👍